tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
Why does this option even exist if everyone just disables it? Do you think keeping it enabled until there is a clear issue is useful? The TS default is true right?

Reply:
In theory it's good because it'll warn us if there's something broken in a lib, but in practice it just becomes noise, and there's not much we can do about broken types in a third party dependency anyway.